### PR TITLE
Continue when getting a redirect from a health check

### PR DESCRIPTION
**Pull request by cdonati**

### CODE REVIEW

File: AppManager/app_manager_server.py [hunk 1]
```diff
@@ -66,6 +66,20 @@ def __init__(self, value):
...
     return repr(self.value)
 
+class NoRedirection(urllib2.HTTPErrorProcessor):
+  """ A url opener that does not automatically redirect. """
+  def http_response(self, request, response):
+    """ Processes HTTP responses.
+
+    Args:
+      request: An HTTP request object.
+      response. An HTTP response object.
```
Comment (menivaitsi) [inline AppManager/app_manager_server.py]: Replace the '.' with ':'

File: AppManager/app_manager_server.py [hunk 2]
```diff
@@ -282,7 +296,11 @@ def wait_on_app(port):
   url = "http://" + private_ip + ":" + str(port) + FETCH_PATH
   while retries > 0:
     try:
-      urllib.urlopen(url)
+      opener = urllib2.build_opener(NoRedirection)
+      response = opener.open(url)
+      if response.code != 200:
```
Comment (menivaitsi) [inline AppManager/app_manager_server.py]: Use a constant for 200

File: AppManager/app_manager_server.py [hunk 3]
```diff
@@ -282,7 +296,11 @@ def wait_on_app(port):
...
+        logging.info('{} returned {}. Headers: {}'.
```
Comment (menivaitsi) [inline AppManager/app_manager_server.py]: Should we have this as a warning instead?

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/717/

**Pull request merged**